### PR TITLE
Translate Arabic text resources

### PR DESCRIPTION
**Pull request by uomar8**
I translated some and still some hopefully soon

### CODE REVIEW

File: main/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">EverTranslator (ترجمة الشاشة)</string>
+    <string name="translation_provider_microsoft_azure">"مايكروسوفت أزور "</string>
+    <string name="translation_provider_google_ml_kit">Google MLKit</string>
+    <string name="translation_provider_google_translate_app">تطبيق ترجمة جوجل</string>
+    <string name="translation_provider_none">استخراج النص فقط</string>
+    <string name="translation_provider_none">"استخراج النص "</string>
```
Comment (firemaples) [inline main/src/main/res/values-ar/strings.xml:8]: The key `translation_provider_none` is listed double times on line 7 and 8.
Please remove one of them, thank you!

Comment (uomar8) [reply main/src/main/res/values-ar/strings.xml]: thank you I don't know how this happened

**Comment by firemaples** [comment]
Thanks for translating the text resources, would you like me to merge it when I have time?

**Comment by uomar8** [comment]
> Thanks for translating the text resources, would you like me to merge it when I have time?

Yes I test it it looks great 

What is missing is 
-the tutorial (because it's old because I think it needs an update)
-language list (there's too many I hope if there's an automatic way and I only check if there's a mistake)

**Comment by firemaples** [comment]
> > Thanks for translating the text resources, would you like me to merge it when I have time?
> 
> Yes I test it it looks great 
> 
> What is missing is 
> -the tutorial (because it's old because I think it needs an update)
> -language list (there's too many I hope if there's an automatic way and I only check if there's a mistake)
> 

For the language list, as you said, I used Bing translator to translate the content of the xml first, then double check if there's any fault.

**Comment by uomar8** [comment]
> > > Thanks for translating the text resources, would you like me to merge it when I have time?
> > 
> > 
> > Yes I test it it looks great
> > What is missing is
> > -the tutorial (because it's old because I think it needs an update)
> > -language list (there's too many I hope if there's an automatic way and I only check if there's a mistake)
> 
> For the language list, as you said, I used Bing translator to translate the content of the xml first, then double check if there's any fault.

Now it's done everything translated and checked 👍

**Comment by firemaples** [comment]
Looks great, thank you for the contribution! 🎉

**Pull request merged**